.bash_prompt

```diff
@@ -57,6 +57,9 @@ else
   sexy_bash_prompt_symbol_color="" # NORMAL
 fi
 
+# Define the default prompt terminator character '$'
+sexy_bash_prompt_symbol='$'
```

Comment:
Yep, definitely escape it then. It looks like that's what we did in the previous implementation.